packages/account/src/Sections/Verification/ProofOfOwnership/proof-of-ownership-form.tsx

```diff
@@ -12,274 +12,288 @@ import FormSubHeader from '../../../Components/form-sub-header';
-                    errors.data[card_key][item_key] = errors.data?.[card_key]?.[item_key] ?? {};
-                    const payment_method = items?.[item_key];
-                    const payment_method_identifier = payment_method?.payment_method_identifier?.trim();
-                    const is_payment_method_identifier_provided =
-                        payment_method?.is_generic_pm || payment_method_identifier?.length > 0;
-                    const is_credit_or_debit_card = payment_method?.identifier_type === IDENTIFIER_TYPES.CARD_NUMBER;
-                    total_documents_uploaded = payment_method?.files?.filter(Boolean)?.length ?? 0;
-                    if (is_payment_method_identifier_provided) {
-                        are_files_uploaded = total_documents_uploaded === payment_method.documents_required;
-                    } else if (
-                        (!payment_method?.documents_required && total_documents_uploaded === 0) ||
-                        (!is_payment_method_identifier_provided && total_documents_uploaded === 0)
-                    ) {
-                        are_files_uploaded = true;
-                    } else if (
-                        (payment_method?.documents_required &&
-                            is_payment_method_identifier_provided &&
-                            total_documents_uploaded === 0) ||
-                        (!is_payment_method_identifier_provided &&
-                            total_documents_uploaded === payment_method?.documents_required * 0.5)
-                    ) {
-                        are_files_uploaded = false;
-                    }
-                    delete errors.data[card_key][item_key].payment_method_identifier;
-                    payment_method?.files?.forEach((file, i) => {
-                        errors.data[card_key][item_key].files = errors?.data?.[card_key]?.[item_key]?.files ?? [];
-                        if (file?.type && !/(image|application)\/(jpe?g|pdf|png)$/.test(file?.type)) {
-                            errors.data[card_key][item_key].files[i] = localize(
-                                "That file format isn't supported. Please upload .pdf, .png, .jpg, or .jpeg files only."
-                            );
-                        }
-                        if (file?.size / 1024 > 8000) {
-                            errors.data[card_key][item_key].files[i] = localize(
-                                'That file is too big (only up to 8MB allowed). Please upload another file.'
-                            );
-                        }
-                        if (errors.data[card_key][item_key].files.length === 0) {
-                            delete errors.data[card_key][item_key].files;
+        const grouped_payment_method_data_keys = Object.keys(grouped_payment_method_data);
+        const initial_values = {};
+        const [form_state, setFormState] = useStateCallback({ should_show_form: true });
```

Comment:
Is this required? I think `is_btn_loading` can be got from the API call or from the formik status?